lib/routes/account.js

```diff
@@ -223,8 +225,16 @@ module.exports = (
                 return log.notifyAttachedServices('verified', request, {
                   email: account.email,
                   uid: account.uid,
-                  locale: account.locale
+                  locale: account.locale,
+                  marketingOptIn: marketingOptIn,
                 })
+              } else if (marketingOptIn) {
+                return marketingCache.set(account.uid, true)
+                  .catch(err => log.error({
```

Comment:
Should we consider moving these `.catch(...)` handlers inside `cache.get` and `cache.set` or would that be going too far?

It seems like, whatever we're using memcached for, we'd never want an error to fail the wider operation.